src/physalis.h

```diff
@@ -37,7 +37,7 @@
  *  DIV_ST
  * USAGE
  */
-#define DIV_ST 1e-10
+#define DIV_ST -1e-10
```

Comment:
Why negative instead of positive? Maybe add a comment about that

Reply:
God, this one is long time ago. What I remember is that positive or negative DIV_ST will make the lebdev nodes are located either slightly farther or closer than particle radius when transforming from r-theta-phi to x-y-z, which then has an effect when the particle is touching the boundary. Initially I thought this has an big impact on temperature boundary conditions because I made a lot of changes there. In the end it turns out to be a small effect. Making it negative doesn't make any sense for those if's in "particle_kernel.cu", for example line 1908. So I think we should keep it positive.